scripts/version/README.md

```diff
@@ -0,0 +1,34 @@
+## How sonar-dotnet version is set in our pipeline
+
+During the pipeline run, we promote the following artifacts:
+
+1. `sonar-csharp-plugin-x.y.z.buildNumber.jar`
+1. `sonar-vbnet-plugin-x.y.z.buildNumber.jar`
+1. `SonarAnalyzer.CSharp-x.y.z.nupkg`
+1. `SonarAnalyzer.VisualBasic-x.y.z.nupkg`
+1. `SonarAnalyzer.CFG-x.y.z.nupkg` (promoted only on repox)
+
+### How is the version set for the *.jar artifacts
+
+The Java artifacts' major, minor and, patch parts of the version, are set in the [sonar-dotnet pom.xml](https://github.com/SonarSource/sonar-dotnet/blob/master/pom.xml#L14).
```

Comment:
This is not entirely true. The actual version for JAR is inside `sonar-csharp-plugin` and `sonar-vbnet-plugin` pom.xml. It should mention that version is updated in all pom.xml files in the repo. See https://github.com/SonarSource/sonar-dotnet/pull/9076/files

Reply:
Isn't the jar versions set from the parent pom that is in the root?
That's my understanding of the `parent` tag [here](https://github.com/SonarSource/sonar-dotnet/blob/master/sonar-vbnet-plugin/pom.xml#L9).

We set this manually, as the `version` tag in the `parent` has to agree with the parent pom to be able to build. But the version is inherited from the parent. [Do I get something wrong here](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#the-scenario)?
I also just tested this in the pipeline.